### ISSUE TITLE
Bug: lib64 not supported

### ISSUE THREAD

**Issue by Teklad** [notepadqq/notepadqq#776]
It looks like we're supporting the common "lib" directory in our notepadqq launch script, but are failing to account for systems that use lib64 install directory by default (mine being one of them).

In order for cmake to do its job properly we will need to support this soon.

**Comment by JuBan1** [comment]
Does this happen when you build from source? If so, I guess QMake's  `$$APPDATADIR` turns out to be lib64 while we have `lib/` hardcoded in the launch script.

I'm a bit peeved that we need a separate nqq launch script and a nqq-bin file to begin with. Other programs manage just fine with just a binary. Why can't we?

**Comment by Teklad** [comment]
Let me rephrase, it still works when using qmake.  However, that's only because we specifically target /usr/lib directly without conforming to the distribution's packaging principles. Arch Linux's /usr/lib directory is actually a symlink to /usr/lib64, which makes it all "just work".  

Our qmake scripts are pretty dated and use a few things that are probably very unfriendly to package maintainers.  I'm gonna go out on a limb here and say one of the reasons we have yet to find a Debian maintainer might be due to this.  They tend to have much stricter standards when it comes to packaging compared to other distributions.

**Comment by JuBan1** [comment]
Word. Our QMake script also doesn't allow any build directories other than notepadqq/out. Or at least I never got it to work. I once tried to "fix" things in the past but at some point I just gave up.

**Comment by noloader** [comment]
@Teklad,

> It looks like we're supporting the common "lib" directory in our notepadqq launch script, but are failing to account for systems that use lib64 install directory by default 

On GNU Linux systems, this is controlled via `config.site`. There are several manual pages that cover using `config.site`:

* [Autoconf, 15.8 Setting Site Defaults](https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Site-Defaults.html)
* [Automake, 2.2.5 Overriding Default Configuration Setting with config.site](https://www.gnu.org/software/automake/manual/html_node/config_002esite.html)

So you should only need to look for a `<prefix>/share/config.site`, like `/usr/share/config.site` or `/usr/local/share/config.site`, and read the value from the configuration file.

In fact, Automake sources the config file so the values are available in the current shell:

> If a file named prefix/share/config.site exists, configure will source it at the beginning of its execution.

-----

> In order for cmake to do its job properly we will need to support this soon.

CMake is likely a separate problem. It does what it wants, and does not follow [GNU Coding Standards](https://www.gnu.org/prep/standards/) on Linux. Ignoring the coding standards on Linux creates a lot of problems (speaking from experience as a maintainer of another library).
